src/terminal/init.lua

```diff
@@ -28,16 +29,21 @@ end
 
 
 local sys = require "system"
-local t -- the terminal/stream to operate on, default io.stderr
-local bsleep  -- a blocking sleep function
-local asleep   -- a (optionally) non-blocking sleep function
-
-
 -- load submodules
```

Comment:
```suggestion
-- load submodules
package.loaded["terminal"] = M  -- cache the empty (for now) module table to prevent require loops
```

see the explanation in